graph/ShortestWay/ShortestWay.h

```diff
@@ -0,0 +1,36 @@
+//
+// Created by emyrmak on 7/9/2019.
+//
+#pragma once
+#include <limits>
+#include <vector>
+#include "Vertices.h"
+
+using Matrix = std::vector<std::vector<int>>;
+using intVector = std::vector<int>;
+using boolVector = std::vector<bool>;
+
+class ShortestWay  {
+public:
+    ShortestWay (Vertices& vertices);
+    ~ShortestWay ();
+    void findShortestWay(int startPoint, int endPoint);
+private:
```

Comment:
it is easier to read if you separate private methods from private variables. You may add the 'private:' keyword twice